www/client/containers/Routes/index.js

```diff
@@ -24,6 +26,8 @@ class Routes extends React.Component {
     return (
       <Switch>
         <Route path='/' exact component={Home} />
+        <Route path='/dataVisualization' exact component={DataVisualizationOverview} />
```

Comment:
Seems like this should be snake case for a url data-visualization

Reply:
Sure